cmake/EthExecutableHelper.cmake

```diff
@@ -139,6 +139,12 @@ macro(eth_install_executable EXECUTABLE)
 			COMPONENT ${EXECUTABLE}
 		)
 
+		install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/RelWithDebInfo/"
+			DESTINATION bin
```

Comment:
The other uses the build type name, like "Release" and "Debug". See the missing "/" in the end. I find a bit awkward. "bin" fixes some DLL install problems (e.g. evmjit.dll is installed to bin).

We should've design the  installed files layout up front. But now I just want something that works.